kolibri_exercise_perseus_plugin/assets/src/views/index.vue

```diff
@@ -257,15 +261,13 @@
             this.$refs.perseusContainer,
             () => {
               this.loading = false;
-              window.ex = this;
```

Comment:
What was this originally for?

Reply:
I was using it for a reference, and forgot to delete it, I think!